### DISCUSSION TITLE
Becoming a contributor

### DISCUSSION THREAD

**Discussion by Jonas2019** [nkaz001/hftbacktest#77] [General]
Hi @nkaz001 , thank you for this awesome project and keep up the great work!

We have talked briefly via Reddit and I really want to become a contributor to this awesome project and let more people know about this!

Please let me know about where the help is needed and we can talk more about the first steps to get started!

**Comment by nkaz001** [comment]
Please see the following items:

-   Improve performance statistics reporting
-   Implement test code
-   Add additional queue or exchange models
-   Update documentation and examples
-   Implement a live bot connector

The tasks you can contribute to may vary depending on your familiarity with different programming languages.

If you're familiar with Python, improving performance statistics reporting and adding test code may be the priority. If you're familiar with Rust, you can add a live bot connector, related features like external connector support through ipz, or npz serialization.

Please contact me through Discord @nkaz001 or reddit DM. Thank you!

**Reply by nkaz001** [reply]
I updated the roadmap. Please see https://github.com/nkaz001/hftbacktest/blob/master/ROADMAP.md
